widgets/search-box/__tests__/search-box-test.js

```diff
@@ -168,6 +168,59 @@ describe('search-box()', () => {
     });
   });
 
+  context('input', () => {
+    let input;
+    beforeEach(() => {
+      container = document.createElement('div');
+      input = createHTMLNodeFromString('<input />');
+      input.addEventListener = sinon.spy();
+    });
+
+    context('instant search', () => {
+      beforeEach(() => {
+        widget = searchBox({container, autofocus: 'auto'});
+        widget.getInput = sinon.stub().returns(input);
+      });
+
+      it('do not perform the search on keyup', () => {
+        // Given
+        helper.state.query = 'foo';
+        // When
+        widget.init(initialState, helper);
+        // Then
+        expect(input.addEventListener.called).toEqual(true);
+        expect(input.addEventListener.args[1].length).toEqual(3);
+        expect(input.addEventListener.args[1][0]).toEqual('input');
```

Comment:
Shouldn't we do two tests, one for `addEventListener` and one for `attachEvent`, based on what the browser is able to do?

Those tests should only test that when triggering an `input` event on the search input, the correct method is called.